pkg/controller/template/render_test.go

```diff
@@ -316,6 +316,12 @@ func TestGenerateMachineConfigs(t *testing.T) {
 	}
 }
 
+func validateSSHConfig(t *testing.T, mc *mcfgv1.MachineConfig) {
```

Comment:
I would move this down with the other helper functions in the bottom of file to stay consistent with file conventions.